vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/UploadScalar.java

```diff
@@ -32,22 +36,34 @@ public Object parseLiteral(Object o) throws CoercingParseLiteralException {
       }).build();
   }
 
+  /**
+   * @deprecated Use the factory method to create a coercing scalar object.
```

Comment:
I don't believe it is necessary to deprecate this method, it isn't changed.

Actually, since the module is in TP, I believe it's fine to skip deprecation altogether.

Just make sure the breaking changes page is updated if this goes to 3.8.4

Reply:
I marked them all deprecated because i noticed that i could get rid of the `Coerce` interface because the static factory returns the right type. Alternatively i can create a implementation, keep the methods and avoid the duplication on the factory method.

Reply:
I understand. My previous comment still stands though: the module is in TP and this is a recently contributed feature, not even merged in 3.8.